src/commands/paranet/paranet-sync-command.js

```diff
@@ -278,20 +265,84 @@ class ParanetSyncCommand extends Command {
                 });
                 return false;
             }
+
+            const data = await this.operationIdService.getCachedOperationIdData(getOperationId);
+
+            const keyword = await this.ualService.calculateLocationKeyword(
```

Comment:
We have assertionId here, maybe pass it as parameter not to make unncesery blockchain call